state-chain/chains/src/lib.rs

```diff
@@ -485,13 +486,27 @@ pub trait ChainEnvironment<
 pub enum SetAggKeyWithAggKeyError {
 	Failed,
 	FinalTransactionExceededMaxLength,
+	DispatchError(DispatchError),
+}
+
+#[derive(RuntimeDebug, Clone, PartialEq, Eq)]
+pub enum SetGovKeyWithAggKeyError {
+	Failed,
```

Comment:
`Failed` is not very descriptive (obviously when there's an error something failed). Can we do better?